src/components/commons/cards/RegisteredCard.tsx

```diff
@@ -32,13 +32,13 @@ export const RegisteredCard = ({ path, postType, title, address, category, creat
   const { multiState, toggleClick } = useMultiState(['removeRegisterdModal']);
```

Comment:
```suggestion
 const {isVisible, handleToggleClick} = useToggleButton();
```
모달을 하나만 사용하시는 거라면 useMultiState 대신에 toggleButton 훅을 사용하셔도 충분할 것 같습니다!

Reply:
네! 변경했습니다~